Authenticator/Source/AppController.swift

```diff
@@ -43,8 +43,15 @@ class AppController {
     }()
 
     init() {
+        if Process.isDemo {
+            store = DemoTokenStore()
```

Comment:
Remind me what issues you were having with the keychain? I think the simulator keychain was fixed in Xcode 8.2.